src/runaqa.ts

```diff
@@ -57,6 +58,9 @@ export async function runaqaTest(
   await exec.exec(
     `${sevenzexe} e ${dependents} -o${process.env.GITHUB_WORKSPACE}/aqa-tests/TKG/lib`
   )
+  if(buildList.includes('system') || buildList.includes('openjdk')) {
```

Comment:
openjdk tests is a different category from system. So we can just move this function call getAqaSystemTestsRepo to the following block https://github.com/adoptium/run-aqa/pull/102/files#diff-22673e27ee79038a5688a185a2cd7775b8d2a63bfb6c4b449d1d5195cc272faaR65

`if (buildList.includes('system')) {`